src/jit/codegencommon.cpp

```diff
@@ -3880,12 +3889,14 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
             }
         }
 
-#ifdef _TARGET_ARM_
-        var_types regType = varDsc->lvIsHfaRegArg ? varDsc->GetHfaType()
-                                                  : varDsc->TypeGet();
-#else // !_TARGET_ARM_
         var_types regType = varDsc->TypeGet();
-#endif // !_TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef could be removed since now we have accessor lvIsHfaRegArg() that returns false on non-hfa targets.